build_sdk.py

```diff
@@ -728,6 +728,7 @@ def build_sel4_config_component(
 def main() -> None:
     parser = ArgumentParser()
     parser.add_argument("--sel4", type=Path, required=True)
+    parser.add_argument("--board", help="Target board")
```

Comment:
I think maybe something like `--single-board` would be better to emphasize that the SDK will only be built for a single board.

Reply:
Or if we wanted we could take a list of boards with something like `--filter-boards zcu102,ultra96v2`

Reply:
I added the option to pass comma separated list of boards as argument `--filter-boards`.
Answering to the previous comment: having the new logic of selecting boards, I did keep the code which creates a new list of chosen boards. Other than that, there were 2 occurrences of loops that went through SUPPORTED_BOARDS, so composing a new list that can be used further as many times as needed seems more reasonable.